sci-physics/fastjet/fastjet-3.4.0.ebuild

```diff
@@ -28,7 +28,7 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 BDEPEND="
-	app-doc/doxygen[dot]
+	doc? ( app-doc/doxygen[dot] )
```

Comment:
To me the use of `app-doc/doxygen[dot]` seems better in case the doxygen ebuild changes. Also <https://gitweb.gentoo.org/repo/gentoo.git/tree/app-doc/doxygen/doxygen-1.9.4-r1.ebuild> lists `media-libs/freetype`.

`DOCS_DEPEND="app-doc/doxygen[dot]"` did not work, probably because it already contains doxygen.

Reply:
> `DOCS_DEPEND="app-doc/doxygen[dot]"` did not work, probably because it already contains doxygen.

That is weird, this should also work. It pulls in doxygen but doesn't pull in graphviz?

> To me the use of `app-doc/doxygen[dot]` seems better in case the doxygen ebuild changes. Also https://gitweb.gentoo.org/repo/gentoo.git/tree/app-doc/doxygen/doxygen-1.9.4-r1.ebuild lists `media-libs/freetype` for `dot`.

Yes but usually having graphviz present is enough to get things working.

Note that the dot flag on doxygen doesn't really do anything. All it does is pull in graphviz and freetype, the installed files are identical (unless USE="doc" is also enabled, and even then USE="dot" is still irrelevant at doxygen runtime). For this reason the doxygen package probably shouldn't have the dot flag in the first place, strictly speaking. Which is why I usually avoid depending on it.